src/main/flight/rpm_filter.c

```diff
@@ -161,34 +163,31 @@ FAST_CODE_NOINLINE void rpmFilterUpdate(void)
         }
 
         // cycle through all notches on ROLL (takes RPM_FILTER_DURATION_S at max.)
-        harmonicIndex = (harmonicIndex + 1) % rpmFilter.numHarmonics;
-        if (harmonicIndex == 0) {
-            motorIndex = (motorIndex + 1) % getMotorCount();
+        //not sure if it correct
```

Comment:
please test.

Reply:
I made some tests on whoop, waiting for the weather to test on 5"

Reply:
Is there a some reason for his change?

Reply:
> Is there a some reason for his change?

I was just trying to simplify.

Reply:
Old code did harmonicIndex as inner loop and motorIndex as outer. New code reverses it.